src/batch/src/executor/expand.rs

```diff
@@ -78,6 +78,21 @@ impl ExpandExecutor {
             yield chunk;
         }
     }
+
+    pub fn create(input: BoxedExecutor, column_subsets: Vec<Vec<usize>>) -> BoxedExecutor {
```

Comment:
Why not name this `new` and return `Self`, like others?

Reply:
Because I want to return BoxedExecutor, so I name it `create` (((((

Reply:
Because I want to return BoxedExecutor, so I name it `create` (((((

Reply:
@liurenjie1024 [told me to use `new` in my previous PR](https://github.com/risingwavelabs/risingwave/pull/4838#discussion_r953400763)

Reply:
Actually, I am using the convention used in frontend. If `create` is not a convention for batch, I will change it. Both LGTM.